experimental/dds/tree/src/test/utilities/TestUtilities.ts

```diff
@@ -283,8 +283,20 @@ export async function setUpLocalServerTestSharedTree(
 			TestDataStoreType,
 			new TestFluidObjectFactory(registry),
 			{
-				summaryOptions: { initialSummarizerDelayMs: 0 },
 				enableOfflineLoad: true,
+				summaryOptions: {
```

Comment:
Consider exposing "standard" config for all tests in one place, and just overwriting 1-2 properties that uniquely needed for specific test. That makes it easier to add new policies in the future - we will not need to change 20 files at once (most likely)

Reply:
great idea!